build.gradle

```diff
@@ -100,6 +99,13 @@ subprojects { project ->
             dependency 'org.springframework:spring-webmvc:4.3.18.RELEASE'
             dependency 'org.testng:testng:6.14.3'
             dependency 'net.javacrumbs.json-unit:json-unit:2.0.0.RC1'
+
+            dependency 'ru.yandex.qatools.matchers:nio-matchers:1.3'
+            dependency 'com.fasterxml.jackson.core:jackson-databind:2.7.0'
+            dependency 'com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.7.0'
+            dependency 'org.apache.tika:tika-core:1.14'
+            dependency 'commons-codec:commons-codec:1.11'
```

Comment:
if this only used for converting bytes to hex string consider removing dependency and using the following util method:

```java
return new BigInteger(1, bytes).toString(16);
```